tests/CriteriaTest.php

```diff
@@ -133,30 +133,4 @@ public function testExpr(): void
     {
         self::assertInstanceOf(ExpressionBuilder::class, Criteria::expr());
     }
-
-    public function testPassingNonOrderEnumToOrderByIsDeprecated(): void
-    {
-        $this->expectDeprecationWithIdentifier('https://github.com/doctrine/collections/pull/389');
-        $criteria = Criteria::create()->orderBy(['foo' => 'ASC']);
-    }
-
-    public function testConstructingCriteriaWithNonOrderEnumIsDeprecated(): void
-    {
-        $this->expectDeprecationWithIdentifier('https://github.com/doctrine/collections/pull/389');
-        $criteria = new Criteria(null, ['foo' => 'ASC']);
-    }
-
-    public function testUsingOrderEnumIsTheRightWay(): void
-    {
-        $this->expectNoDeprecationWithIdentifier('https://github.com/doctrine/collections/pull/389');
-        Criteria::create()->orderBy(['foo' => Order::Ascending]);
-        new Criteria(null, ['foo' => Order::Ascending]);
-    }
-
-    public function testCallingGetOrderingsIsDeprecated(): void
-    {
-        $criteria = Criteria::create()->orderBy(['foo' => Order::Ascending]);
-        $this->expectDeprecationWithIdentifier('https://github.com/doctrine/collections/pull/389');
-        $criteria->getOrderings();
-    }
```

Comment:
Even if you take back the deprecation, those execution paths still have to be tested.